injector/src/main/java/edu/ucr/cs/riple/injector/location/OnField.java

```diff
@@ -45,7 +45,7 @@ public OnField(String uri, String clazz, Set<String> variables) {
 
   @Override
   public Location duplicate() {
-    return new OnField(clazz, uri, new HashSet<>(variables));
+    return new OnField(uri, clazz, new HashSet<>(variables));
```

Comment:
Interesting find... was this spotted by an Error Prone check?

Reply:
Yes :) but this method is not used anymore.

Reply:
Then why do we keep it, rather than cleaning it up altogether? Or is it just not used in the default configuration but still something we want to support?

Reply:
Actually this method is still called in the tests, there was a usage for this in previous versions of `injector` where we had a flag to distinguish `add` from `remove` annotation request. But at this moment, we have a clear types of `RemoveAnnotation` and `AddAnnotation` that extends the `Change` class. With this design we no longer use `duplicate()` method in `core` module to remove the injected annotations. However, this method is still used in a test. (The test checks we do not inject multiple `@Nullable` annotations on the same element once one is already injected)

Reply:
And yet nothing was broken with that test due to the parameters being reversed here? Um... either way, agree at this point that this is separate from this PR at any rate :)